README.md

```diff
@@ -1,14 +1,67 @@
-# Autometrics Go
+![GitHub_headerImage](https://user-images.githubusercontent.com/3262610/221191767-73b8a8d9-9f8b-440e-8ab6-75cb3c82f2bc.png)
+
+[![Go Reference](https://pkg.go.dev/badge/github.com/autometrics-dev/autometrics-go.svg)](https://pkg.go.dev/github.com/autometrics-dev/autometrics-go)
+[![Discord Shield](https://discordapp.com/api/guilds/950489382626951178/widget.png?style=shield)](https://discord.gg/kHtwcH8As9)
+
+Metrics are a powerful and cost-efficient tool for understanding the health and
+performance of your code in production. But it's hard to decide what metrics to
+track and even harder to write queries to understand the data.
 
 Autometrics is a [Go
 Generator](https://pkg.go.dev/cmd/go#hdr-Generate_Go_files_by_processing_source)
-bundled with a library that instruments your functions and gives direct links to 
-inspect usage metrics from your code.
+bundled with a library that instruments your functions with the most useful
+metrics: request rate, error rate, and latency. It standardizes these metrics
+and then generates powerful Prometheus queries based on your function details to
+help you quickly identify and debug issues in production.
+
+## Benefits
+
+- ✨ `//autometrics:inst` directive adds useful metrics to any function, without you thinking about what metrics to collect
+- 💡 Generates powerful Prometheus queries to help quickly identify and debug issues in production
+- 🔗 Injects links to live Prometheus charts directly into each function's doc comments
+- [📊 Grafana dashboards](https://github.com/autometrics-dev/autometrics-shared#dashboards) work without configuration to visualize the performance of functions & [SLOs](https://docs.rs/autometrics/latest/autometrics/objectives/index.html)
+- 🔍 Correlates your code's version with metrics to help [identify commits](https://fiberplane.com/blog/autometrics-rs-0-4-spot-commits-that-introduce-errors-or-slow-down-your-application) that introduced errors or latency
+- 📏 Standardizes metrics across services and teams to improve debugging
+- ⚖️ Function-level metrics provide useful granularity without exploding cardinality
+
+## Advanced Features
+
+- [🚨 Define alerts](#generate-alerts-automatically) using SLO best practices directly in your source code
+- [📍 Attach exemplars](#exemplar-support) automatically to connect metrics with traces
+- [⚙️ Configurable](#opentelemetry-support) metric collection library
+  ([`opentelemetry`](https://pkg.go.dev/go.opentelemetry.io/otel) or
+  [`prometheus`](https://pkg.go.dev/github.com/prometheus/client_golang))
+
+See [autometrics.dev](https://docs.autometrics.dev/) for more details on the ideas behind autometrics.
+
+**Table of Contents**
```

Comment:
toc is integrated in the github ui already

![image](https://github.com/autometrics-dev/autometrics-go/assets/24739711/0e391669-8c95-40e1-b5cf-ea880f6b7f79)